PLCLocation.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Collection.Generic;
+
+
+namespace mujinplccs{
+
+    public class Order{
+        public Order(){}
+        // some order finish status
+        public PLCLogic.PLCOrderCycleFinishCode orderCycleFinishCode;
+        public PLCLogic.PLCPreparationCycleFinishCode preparationCycleFinishCode;
+
+        // customer implemented interface
+        abstract public void onFinishCodeChangedCallback();
+        abstract public void onNumberPutInDestChangedCallback();
+        // order info
+        public string partType{get; set;}
+        public int orderNumber{get; set;}
+        public string uniqueId{get; set;}
+        public string containerId{get; set;}
+        public string containerType{get; set;}
+        public string robotId{get; set;}
+        public string placeContainerId{get; set;}
+        public string placeContainerType{get; set;}
+        public PLCLocation placeLocation{get; set;}
+        public PLCLocation pickLocation{get; set;}
```

Comment:
For current particular case it is singular, but I think we do support multiple (although it might generate confusion)